docs/source/tutorial_usb_analysis.rst

```diff
@@ -0,0 +1,131 @@
+Protocol analysis of a USB keyboard
+===================================
+
+This tutorial walks through the whole process of running a USB protocol capture of a target device, in this case a keyboard. Hopefully most people have a USB keyboard available and can follow along,
+though this process is also applicable to any USB peripheral device.
+
+Prerequisites
+-------------
+
+ * Install the Cynthion tools by following :doc:`getting_started`
+ * Install Packetry by following `Getting Started with Packetry <https://packetry.readthedocs.io/en/latest/quick_start.html>`_
+ * Run the analyzer gateware on Cynthion by following :doc:`getting_started_packetry`
+
+Determine device speed
+----------------------
+
+USB 2.0 supports three different speeds: Low (1.5 Mbit/s), Full (12 Mbit/s), and High (480 Mbit/s).
+The analyzer needs to know what speed to expect, so we need to detemine what speed the target device is using.
+
+    .. note::
+
+        Soon this step won't be necessary, as the analyzer will be able to determine the speed automatically,
+        but that feature is currently in development.
+
+To determine the speed, we plug the target device into a host check what speed it reports. The way to check depends on the host operating system:
+
+.. tab:: Linux
+
+    Run the command ``sudo dmesg -W`` in a terminal window and then plug the target device in, some new lines should show up with information about it::
+
+        [975321.743878] usb 1-6.1: new full-speed USB device number 12 using xhci_hcd
+        [975321.821329] usb 1-6.1: New USB device found, idVendor=3434, idProduct=0108, bcdDevice= 2.00
+        [975321.821341] usb 1-6.1: New USB device strings: Mfr=1, Product=2, SerialNumber=0
+        [975321.821345] usb 1-6.1: Product: Keychron Q1
+        [975321.821348] usb 1-6.1: Manufacturer: Keychron
+
+    Here, it shows that the target device has enumerated at full-speed.
+
+
+.. tab:: macOS
+
+    Go to the Apple menu -> About This Mac -> Hardware -> USB. Highlight the target device and check the Speed field.
```

Comment:
On my test system (macOS 14) I had to go to: Apple menu -> About This Mac -> More Info -> System Report -> Hardware -> USB